core/includes/common.inc

```diff
@@ -2016,7 +2032,8 @@ function _filter_xss_attributes($attr) {
             strlen($attrname) > 96
           );
           $working = $mode = 1;
-          $attr = preg_replace('/^[-a-zA-Z]+/', '', $attr);
+          // @todo: fix the regex properly in https://github.com/backdrop/backdrop-issues/issues/6575
```

Comment:
Let's make sure we get that in before this one :) and then remove this

Reply:
That was the plan, yes 👍🏼 🙂